### PR TITLE
Updated all dependencies for KNIME 3.6 and 3.7

### PR DESCRIPTION
**Pull request by manuelschwarze**
Updated all MANIFEST.MF files to refer only to plugins available in KNIME
3.6 and 3.7 to ensure full compatibility.
Fixed internal RDKit plugins dependencies to be fully in sync with each
other.
Increased RDKit plugin / feature version to 3.6.1.
Updated feature copyrights to 2019.

### CODE REVIEW

File: org.rdkit.knime.binaries.feature/feature.xml [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
+      version="3.6.1.qualifier"
       provider-name="NIBR">
 
    <description>
       This feature contains RDKit binaries for use as dependencies for other packages.
    </description>
 
    <copyright>
-      Copyright (C) 2015 Novartis Institutes for BioMedical Research
+      Copyright (C) 2019 Novartis Institutes for BioMedical Research
```
Comment (greglandrum) [inline org.rdkit.knime.binaries.feature/feature.xml]: I think it's better to do this as: `Copyright (C) 2015-2019` in order to indicate the full timeline.

**Review by greglandrum — CHANGES_REQUESTED** [review]
I'd change the way the copyright dates have been updated, but otherwise looks good.

**Comment by manuelschwarze** [comment]
Then it has to be 2012-2019

-Manuel

Sent from my iPhone

Am 12.05.2019 um 07:24 schrieb Greg Landrum <notifications@github.com<mailto:notifications@github.com>>:


@greglandrum requested changes on this pull request.

I'd change the way the copyright dates have been updated, but otherwise looks good.

________________________________

In org.rdkit.knime.binaries.feature/feature.xml<https://github.com/rdkit/knime-rdkit/pull/61#discussion_r283119930>:

>        provider-name="NIBR">

    <description>
       This feature contains RDKit binaries for use as dependencies for other packages.
    </description>

    <copyright>
-      Copyright (C) 2015 Novartis Institutes for BioMedical Research
+      Copyright (C) 2019 Novartis Institutes for BioMedical Research


I think it's better to do this as: Copyright (C) 2015-2019 in order to indicate the full timeline.

**Comment by greglandrum** [comment]
Ok; I was just going from what was there before. The usual thing with copyright dates (I think) is to just extend the second (later) date when a “significant” change is made

File: org.rdkit.knime.binaries.feature/feature.xml (hunk 1 above)
Comment (manuelschwarze) [reply org.rdkit.knime.binaries.feature/feature.xml]: Ok, I put now the initial year as first and the last change year as second.

**Pull request merged**